Templates/Templates/AutoGenerateProtocol.stencil

```diff
@@ -0,0 +1,32 @@
+
+//: Do not change this code as it is autogenerated every time you build.
```

Comment:
We already have warning, don't think there is need to state this in template too.